### ISSUE TITLE
python3: can't open file 'synapse.app.homeserver'

### ISSUE THREAD

**Issue by sylflo** [AVENTER-UG/docker-matrix#31]
Starting from version 0.99.4,

There is an error by launching the command generate

```
docker run -v /tmp/data:/data --rm \
  -e SERVER_NAME=example.com \
  -e REPORT_STATS=yes \
  avhost/docker-matrix:v0.99.4 \
  generate
-=> generate turn config
-=> generate synapse config
python3: can't open file 'synapse.app.homeserver': [Errno 2] No such file or directory
-=> configure some settings in homeserver.yaml
awk: cannot open /data/homeserver.tmp (No such file or directory)
-=> configure some settings in example.com.log.config
sed: can't read /data/example.com.log.config: No such file or directory

-=> you have to review the generated configuration file homeserver.yaml
```

**Comment by mvgorcum** [comment]
Yes, there's a `-m` missing. It should be fixed with #30, or if that PR isn't merged I'll create a separate PR.

The easiest workaround for now is probably to pull an old non-python3 version to run the generate command, and then use 0.99.4.

**Comment by mvgorcum** [comment]
This issue can be closed.
